### PR TITLE
Added ability to teleport straight to warp

### PR DESCRIPTION
**Pull request by ItchyDitchy**

### CODE REVIEW

**Review by OmerBenGera — CHANGES_REQUESTED** [review]
Instead of copying code from MenuWarps, make a static method called "simulateClick" that takes the player, the warp and all the other information that is needed and make both files to use it.

File: src/main/java/com/bgsoftware/superiorskyblock/commands/CmdWarp.java [hunk 1]
```diff
@@ -8,69 +8,91 @@
...
+		Pair<Island, SuperiorPlayer> arguments = args.length == 1 ? CommandArguments.getSenderIsland(plugin, sender) :
+			CommandArguments.getIsland(plugin, sender, args[1]);
+
+		Island island = arguments.getKey();
+		if(island == null)
+			return;
+
+		SuperiorPlayer superiorPlayer = plugin.getPlayers().getSuperiorPlayer(sender);
+		if (args.length == 3)
+			for (String warp : island.getIslandWarps().keySet()) {
```
Comment (OmerBenGera) [inline src/main/java/com/bgsoftware/superiorskyblock/commands/CmdWarp.java:71]: You can use island.getWarp(String) to find a warp by it's name

File: src/main/java/com/bgsoftware/superiorskyblock/commands/CmdWarp.java [hunk 2]
```diff
@@ -8,69 +8,91 @@
...
+	public void execute(SuperiorSkyblockPlugin plugin, CommandSender sender, String[] args) {
+		Pair<Island, SuperiorPlayer> arguments = args.length == 1 ? CommandArguments.getSenderIsland(plugin, sender) :
+			CommandArguments.getIsland(plugin, sender, args[1]);
+
+		Island island = arguments.getKey();
+		if(island == null)
+			return;
+
+		SuperiorPlayer superiorPlayer = plugin.getPlayers().getSuperiorPlayer(sender);
+		if (args.length == 3)
```
Comment (OmerBenGera) [inline src/main/java/com/bgsoftware/superiorskyblock/commands/CmdWarp.java:70]: What if you run /is warp <warp-name>? It sounds like a valid command, but according to the code you must use /is warp <island> <warp-name>, which is not necessary.

**Review by OmerBenGera — CHANGES_REQUESTED** [review]

File: src/main/resources/lang/en-US.yml [hunk 3]
```diff
@@ -106,7 +106,6 @@ COMMAND_ARGUMENT_ENTITY: 'entity'
 COMMAND_ARGUMENT_ISLAND: 'island'
 COMMAND_ARGUMENT_ISLAND_NAME: 'island-name'
 COMMAND_ARGUMENT_ISLAND_ROLE: 'island-role'
-COMMAND_ARGUMENT_ISLAND_WARP 'island-warp'
```
Comment (OmerBenGera) [inline src/main/resources/lang/en-US.yml]: Is there a reason why you removed this line?

**Review by OmerBenGera — CHANGES_REQUESTED** [review]

**Comment by ItchyDitchy** [comment]
> You removed all messages of the modules, why is that?
> Keep it them in the files please.

File: src/main/java/com/bgsoftware/superiorskyblock/commands/CmdWarp.java (hunk 2 above)
Comment (ItchyDitchy) [reply src/main/java/com/bgsoftware/superiorskyblock/commands/CmdWarp.java:70]: I'll go add a setting at config then I guess if they should prioritize the warp or the player.

File: src/main/java/com/bgsoftware/superiorskyblock/commands/CmdWarp.java (hunk 1 above)
Comment (ItchyDitchy) [reply src/main/java/com/bgsoftware/superiorskyblock/commands/CmdWarp.java:71]: Will implement...

**Comment by ItchyDitchy** [comment]
.

File: src/main/resources/lang/en-US.yml (hunk 3 above)
Comment (ItchyDitchy) [reply src/main/resources/lang/en-US.yml]: Probably a mistake

**Comment by OmerBenGera** [comment]
Any news regarding this?

**Pull request closed without merging**